src/site/content/en/blog/responsive-web-design-basics/index.md

```diff
@@ -0,0 +1,579 @@
+(for example, between phones and tablets, and even between different phones),
+content should not rely on a particular viewport width to render well.
+
+In the past, this required setting elements used to create layout in percentages.
+In the example below, you can see a two-column layout with floated elements, sized using pixels.
+Once the viewport becomes smaller than the total width of the columns, we have to scroll horizontally
+to see the content.
+
+<figure class="w-figure">
+  <img src="./fixed-width-layout.jpg" alt="Screenshot of a two-column layout with most of the second column outside the viewport">
+  <figcaption class="w-figcaption">
+    A floated layout using pixels. <a href="https://layout-floats-px.glitch.me/">See this example on Glitch</a>.
+  </figcaption>
+</figure>
+
+By using percentages for the widths, the columns always remain a certain percentage of the container.
+This means that the columns become narrower, rather than creating a scrollbar.
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/layout-floats-percent?path=README.md&previewSize=100"
+    title="layout-floats-percent on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+Modern CSS layout techniques such as Flexbox, Grid Layout, and Multicol
+make the creation of these flexible grids much easier.
+
+#### Flexbox {: #flexbox }
+
+This layout method is ideal when you have a set of items of different sizes
+and you would like them to fit comfortably in a row or rows,
+with smaller items taking less space and larger ones getting more space.
+
+```css
+.items {
+  display: flex;
+  justify-content: space-between;
+}
+```
+
+In a responsive design, you can use Flexbox to display items as a single row,
+or wrapped onto multiple rows as the available space decreases.
+
+<div class="glitch-embed-wrap" style="height: 220px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/responsive-flexbox?previewSize=100"
+    title="responsive-flexbox on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+[Read more about Flexbox](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Flexbox).
+
+#### CSS Grid Layout {: #grid }
+
+CSS Grid Layout allows for the straightforward creation of flexible grids.
+If we consider the earlier floated example,
+rather than creating our columns with percentages,
+we could use grid layout and the `fr` unit,
+which represents a portion of the available space in the container.
+
+```css
+.container {
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+}
+```
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/two-column-grid?previewSize=100"
+    title="two-column-grid on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
```

Comment:
Same cognitive dissonance as the percentage float example:

![image](https://user-images.githubusercontent.com/4713486/82074342-7bd0e980-968f-11ea-8c97-2a82a7d53aa2.png)

Reply:
same issue, now fixed.